server/src/main/java/org/elasticsearch/index/get/GetResult.java

```diff
@@ -393,15 +428,14 @@ public void readFrom(StreamInput in) throws IOException {
             if (source.length() == 0) {
                 source = null;
             }
-            int size = in.readVInt();
-            if (size == 0) {
-                fields = emptyMap();
+            if (in.getVersion().onOrAfter(Version.V_8_0_0)) {
```

Comment:
I think this should be 7.3.0, since we intend to backport to 7.x?

Reply:
at the moment of writing this, there is only 7.2.0 https://github.com/elastic/elasticsearch/blob/master/server/src/main/java/org/elasticsearch/Version.java#L110 
Should we add 7.3.0 to that file?

Reply:
Sorry I mispoke. 7.2.0 is good.

Reply:
hm, what is the regular sequence of actions to create a downport here?

As expected, without adjustments in serialization for 7.2 the tests for the mixed cluster fail. Should I open a similar downport PR against https://github.com/elastic/elasticsearch/tree/7.x, get it merged, and then expect the build here to be green?

Reply:
We handle this by temporarily disabling bwc tests. I just pushed the disabling to your branch. Tomorrow I will test the bwc behavior locally before pushing.

Reply:
@rjernst Looks like 7.2 has already feature freeze. Should this PR target 7.3 and master?

Reply:
Yes, 7.2 was branched, but the 7.3 constant hasn't been pushed yet. I will update this branch today once the constant exists.